src/link.cpp

```diff
@@ -1559,7 +1559,7 @@ static void construct_linker_job_macho(LinkJob *lj) {
             lj->args.append("-lgcc_s.10.5");
         }
     } else {
-        zig_panic("TODO");
+        //zig_panic("TODO");
```

Comment:
You can just delete this entire else branch. In fact, I think we should drop support for 10.5 and 10.6 as well, feel free to delete this entire if statement.

Reply:
Went ahead and got rid of the entire if statement